src/test/java/com/heatheryou/httpserver/RequestParserTest.java

```diff
@@ -2,28 +2,56 @@
 
 import org.junit.Test;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
 import java.util.Arrays;
 import java.util.List;
 
+import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class RequestParserTest {
     @Test
-    public void ParserReturnsResponseLineGivenRequestLine() {
+    public void ItReturnsRequestInListForm() throws IOException {
```

Comment:
Each of these tests is a method in the test class `RequestParserTest`, so they should follow camelcase rather than titlecase (i.e. `itReturnsRequestInListForm()` with the first letter uncapitalized).

Reply:
👍I wasn't too sure what the convention was even after looking it up, so thanks for making this clear!

Reply:
Fixed in c79b1ef4ae234d6f8a8f21934be0061355a03ffc